RULES.md

```diff
@@ -275,3 +275,27 @@ Description: Checks that that the ingress permissions on an EC2 Security Group a
 
 Trigger Type: ```Change Triggered```
 Required Parameter: ```None```
+
+### 32. Check that no EC2 is publicly accessible except 80 and 443.
+Description: Check that no security groups allow public access to the ports other then 80 and 443.
+
+	python/ec2_sg_public_ingress_excluding_80_443.py
+
+Trigger Type: ```Change Triggered```
+Required Parameter: ```None```
+
+### 33. Check that no EC2 are in Public Subnet.
+Description: Check that no EC2 are in Public Subnet.
+
+	python/ec2_vpc_public_subnet.py
+
+Trigger Type: ```Change Triggered```
+Required Parameter: ```None```
+
+### 34. Check that no RDS are in Public Subnet.
```

Comment:
Changes are also added in the relevant python files in comments sections. 
https://github.com/PriceBoardIn/aws-config-rules/commit/bed2c91449a6029b15e01e1db99f87be5c23159b
https://github.com/PriceBoardIn/aws-config-rules/commit/bbb28cf47c9d5cb8f2d2d3216a6c7ae7a1fe0626
https://github.com/PriceBoardIn/aws-config-rules/commit/f9de1356c4b743bbfa5e4e124a4400e700398c17